samwise.js

```diff
@@ -1,9 +1,8 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?